src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  constructor(name, health = 100, hidden = false) {
```

Comment:
The constructor of the Animal class should not have a 'hidden' parameter since not all animals will have this property. It should be specific to the Herbivore class.